data/events.ts

```diff
@@ -70,9 +71,16 @@ export const eventTypes = [
     icon: "flag.svg",
     active: true,
   },
+  {
+    name: EventKind.E1,
+    description: 'ACM Cyber\'s Engineering 1 class, Malware Defense, teaches fundamental cybersecurity concepts and lets you apply what you\'ve learned through a hands-on project: creating your own modular malware! Topics covered include cybersecurity and ethics, phishing, Python, Git, steps of the cyber kill chain, and more. This course is beginner-friendly, so even if you\'ve never coded before, you are welcome to enroll!',
```

Comment:
Maybe add https://l.acmcyber.com/e1-pte as a link - this will point to our form which has instructions on enrollment